Problem1

```diff
@@ -0,0 +1,32 @@
+class Solution {
+    public List<Integer> findDisappearedNumbers(int[] nums) {
+        List<Integer> result = new ArrayList<>();
```

Comment:
Good Job! Please try to add related information as per the sample file provided. Good comments.